Test/MyTaskTest.cs

```diff
@@ -443,6 +443,7 @@ public static void TestFileOutput()
 
             SearchTask task = Toml.ReadFile<SearchTask>(Path.Combine(TestContext.CurrentContext.TestDirectory, @"SlicedSearchTaskConfig.toml"), MetaMorpheusTask.tomlConfig);
             task.SearchParameters.DecoyType = DecoyType.None;
+            task.SearchParameters.WriteMzId = true;
```

Comment:
Is there a test where this is false?

Reply:
Happy to make further updates but i need to understand better what you're looking for. Apparently i haven't had enough coffee yet.

Reply:
I think it's hit... feel free to ignore. I was trying to figure out why codecov had gone down, but I think it's just because of how it's handling the conditionals.